nextcord/raw_models.py

```diff
@@ -1,7 +1,8 @@
 """
 The MIT License (MIT)
 
-Copyright (c) 2015-present Rapptz
+Copyright (c) 2015-2021 Rapptz
+Copyright (c) 2021-present tag-epic
```

Comment:
This date is inconsistent with other files in this pr

Reply:
@MaskDuck you marked this as resolved without
- discussion
- commits

Reply:
the file is edited in 2021